src/stan/mcmc/hmc/nuts/base_nuts.hpp

```diff
@@ -152,14 +152,16 @@ namespace stan {
         names.push_back("stepsize__");
         names.push_back("treedepth__");
         names.push_back("n_leapfrog__");
-        names.push_back("n_divergent__");
+        names.push_back("divergent__");
+        names.push_back("E__");
```

Comment:
CmdStan already passes and there’s a separate PR for updated doc.
So are we merging or what?

On Mar 10, 2016, at 3:27 PM, Daniel Lee <notifications@github.com> wrote:

> In src/stan/mcmc/hmc/nuts/base_nuts.hpp:
> 
> > @@ -152,14 +152,16 @@ namespace stan {
> >          names.push_back("stepsize__");
> >          names.push_back("treedepth__");
> >          names.push_back("n_leapfrog__");
> > -        names.push_back("n_divergent__");
> > +        names.push_back("divergent__");
> > +        names.push_back("E__");
> Summary:
> 
> n_divergent__ moves to divergent__
> E__ is inserted
> I just need to know what sort of impact this has. It's easy for CmdStan.
> 
> —
> Reply to this email directly or view it on GitHub.
>